### PR TITLE
Use encodeName function to each objectName

### PR DESCRIPTION
**Pull request by RRozak**
It makes https://github.com/chipsalliance/UHDM-integration-tests/pull/627 passed and allows to parse tl_*pkg packages by uhdm-verilator.

### CODE REVIEW

File: src/UhdmAst.cpp [hunk 1]
```diff
@@ -44,10 +44,29 @@ void visit_one_to_one(const std::vector<int> childrenNodeTypes, vpiHandle parent
     }
 }
 
+std::string encodeObjectName(std::string name) {
+    // Verilator uses encodeName function on identificators
```
Comment (kbieganski) [inline src/UhdmAst.cpp]:
```suggestion
    // Verilator uses encodeName function on identifiers
```

File: src/UhdmAst.cpp [hunk 2]
```diff
@@ -44,10 +44,29 @@ void visit_one_to_one(const std::vector<int> childrenNodeTypes, vpiHandle parent
...
+    // System functions names are not handled as identificators
```
Comment (kbieganski) [inline src/UhdmAst.cpp]:
```suggestion
    // System functions names are not handled as identifiers
```

**Review by kbieganski — CHANGES_REQUESTED** [review]

File: src/UhdmAst.cpp (hunk 1 above)
Comment (RRozak) [reply src/UhdmAst.cpp]: Done

File: src/UhdmAst.cpp (hunk 2 above)
Comment (RRozak) [reply src/UhdmAst.cpp]: Done

**Review by kbieganski — APPROVED** [review]

**Pull request merged**